recipe/meta.yaml

```diff
@@ -8,118 +8,71 @@ package:
-      requires:
-        - pip
-      files:
-        - check-for-warnings.py
-        - allowed-warnings-base.txt
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - cython
+    - pip
+    - numpy {{ numpy }}
+    - versioneer 0.28
+    - tomli
+    - setuptools >=48.0.0
+    # mkl-service is only available for linux-64, win-64, and osx-64.
+    # For everything else, use openblas.
+    - mkl-service  # [not ((osx and x86_64) or linux and (aarch64 or ppc64le))]
+    - openblas  # [(osx and x86_64) or linux and (aarch64 or ppc64le)]
+    # We also need the "blas" package. Otherwise we get reports
+    # of "cblas.h: No such file or directory" at import time.
+    # Ref: <https://github.com/conda-forge/aesara-feedstock/issues/80>
+    - blas
+    - wheel
+  run:
+    - python
+    - {{ pin_compatible('numpy') }}
+    - scipy >=0.14
+    - filelock
+    - etuples
+    - logical-unification
+    - miniKanren
+    - cons
+    - typing_extensions
+    - mkl-service  # [not ((osx and x86_64) or linux and (aarch64 or ppc64le))]
+    - openblas     # [(osx and x86_64) or linux and (aarch64 or ppc64le)]
 
-  - name: pytensor
-    build:
-      script:
-        - echo "Nothing to build here, just add dependencies."
-    requirements:
-      build:
-        - {{ compiler('c') }}
-        - {{ compiler('cxx') }}
-      host:
-        - python
-      run:
-        - python
-        - {{ pin_subpackage('pytensor-base', exact=True) }}
-        - {{ compiler('c') }}  # [not win]
-        - {{ compiler('cxx') }}  # [not win]
-        - gxx  # [linux]
-        - {{ compiler('m2w64_c') }}  # [win]
-        - {{ compiler('m2w64_cxx') }}  # [win]
-
-        - m2w64-toolchain  # [win]
-        # The libpython conda-forge package seems to be a wrapper package
-        # which configures m2w64-toolchain for Python.
-        # <https://github.com/conda-forge/libpython-feedstock/blob/main/recipe/bld.bat>
-        - libpython >=2.2  # [win]
-
-        # mkl-service is only available for linux-64, win-64, and osx-64.
-        # For everything else, use openblas.
-        - mkl-service  # [linux64 or win64 or (osx and not arm64)]
-        - openblas  # [not (linux64 or win64 or (osx and not arm64))]
-        # For some reason we also need the "blas" package. Otherwise we get reports
-        # of "cblas.h: No such file or directory" at import time.
-        # Ref: <https://github.com/conda-forge/aesara-feedstock/issues/80>
-        - blas
-    test:
-      imports:
-        - pytensor
-        - pytensor.configdefaults
-      commands:
-        - pytensor-cache help
-        - pip check
-        - python check-for-warnings.py allowed-warnings-main.txt
-      requires:
-        - pip
-      files:
-        - check-for-warnings.py
-        - allowed-warnings-main.txt
+test:
+  imports:
+    - pytensor
+    - pytensor.configdefaults
+  commands:
+    - pytensor-cache help
```

Comment:
Could we include source_files `tests` and run `pytest`? At least, for checking if the package has any issues we are not aware yet

Reply:
I will look into this.

Reply:
interestingly, one module for running the tests seems to be missing from the upstream project. Maybe I'm missing some dependency. The module is `tests.unittests`.
For now I will include one test which passes as it doesn't require the unit test module.

Reply:
I think I found the test dependencies! Nevermind....